<launch>
  <node pkg="husky_waypoint_nav" type="optitrack_waypoint_navigator.py" name="optitrack_waypoint_navigator">
    <param name="outdoor_waypoint_nav/coordinates_file" value="$(find husky_waypoint_nav)/config/optitrack_waypoints.txt" />
    <param name="outdoor_waypoint_nav/loop_mode" value="true" />
    <param name="outdoor_waypoint_nav/pause_button_num" value="3" />
    <param name="outdoor_waypoint_nav/resume_button_num" value="1" />
    <param name="outdoor_waypoint_nav/package_path" value="$(find husky_waypoint_nav)/config" />
  </node>
</launch>

